README.md

```diff
@@ -1 +1,10 @@
-# myfirst-microproject
\ No newline at end of file
+# myfirst-microproject
+> This is my first Microverse Project which is a greeting message in the web.
+## Built With 
+* HTML and CSS
+## Author
+* Github: @angeluray
```

Comment:
Please make sure you follow the README template and have a header with your name as well as linking your social media profiles properly.